gossip/tx_scrambler_test.go

```diff
@@ -600,6 +601,51 @@ func TestTxScrambler_FilterAndOrderTransactions_RandomInput(t *testing.T) {
 	}
 }
 
+func TestGetExecutionOrder_TransactionsFiltering(t *testing.T) {
+	tests := []struct {
+		name        string
+		isSonic     bool
+		entries     []ScramblerEntry
+		expectedLen int
+	}{
+		{
+			name:    "TxScramblerIsNotUsed",
+			isSonic: false,
+			entries: []ScramblerEntry{
+				&dummyScramblerEntry{
+					hash: common.Hash{1},
+				},
+				&dummyScramblerEntry{
+					hash: common.Hash{1},
+				},
+			},
```

Comment:
nit: this input is the same for all cases; Thus, I would move it into the test loop below.